internal/pkg/store/secretsmanager/secretsmanager.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package secretsmanager wraps AWS SecretsManager API functionality.
+package secretsmanager
+
+import (
+	// "fmt"
+
+	// "github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/identity"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/service/secretsmanager/secretsmanageriface"
+)
+
+// SecretsManager is in charge of fetching and creating projects, environment and pipeline
+// configuration in SecretsManager.
+type SecretsManager struct {
+	secretsManager secretsmanageriface.SecretsManagerAPI
+	// identity      identityService
+	sessionRegion string
+}
+
+// NewSecretsManager returns a SecretsManager configured with the input session.
+func NewStore() (*SecretsManager, error) {
+	sess, err := session.Default()
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &SecretsManager{
+		secretsManager: secretsmanager.New(sess),
+		// identity:      identity.New(sess),
+		sessionRegion: *sess.Config.Region,
```

Comment:
Do we need to store the region? it doesn't seem to be used

Reply:
Probably not. I was mostly following other clients -- probably don't need identity/region in the current pipeline init workflow but wasn't sure if we might need it in the future?